include/backend/vs/xml.h

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
if I'm not mistaken, Xcode files are in XML too, but I may be wrong.
For VS project, i need to create XML flles with only `<tag attr=val> .... </tag>` and `<tag>val</tag>`

no reader. So it's not so generic. If XCode needs XML files, yes it could be moved.

Reply:
it's difficult to find doc on Xcode project files, but it seems it's not XML: https://pewpewthespells.com/blog/xcconfig_guide.html